frontend/src/app/organization/organization.service.ts

```diff
@@ -19,11 +19,13 @@ export class OrganizationService {
   private organizationEvent = new BehaviorSubject<Organization | null>(null);
   private organizationsEvent = new Subject<Organization[]>();
   private offersEvent = new Subject<Offer[]>();
+  private createOrganizationEvent = new Subject<ContactStatus>();
 
   public contactStatus$: Observable<ContactStatus> = this.contactStatusEvent.asObservable();
   public organization$: Observable<Organization | null> = this.organizationEvent.asObservable();
   public organizations$: Observable<Organization[]> = this.organizationsEvent.asObservable();
   public offers$: Observable<Offer[]> = this.offersEvent.asObservable();
+  public createStatus$: Observable<ContactStatus> = this.createOrganizationEvent.asObservable();
```

Comment:
I don't think createStatus$ is a proper name for that observable - it is misleading. ContactStatus$ was named like that, because it actually refered to the 'status' property of contact form. Please take a look at the names of other obesrvables here.

More proper name would be createOrganization$ or createOrEditOrganization$ in my opinion

Reply:
Also, the type for observable is wrong. It is not "ContactStatus". Please define a new interface in organization.interfaces. Please follow my comments in createOrganization() below.